src/SASLib.jl

```diff
@@ -912,11 +912,11 @@ function createstrarray(handler, column_symbol, nrows)
     elseif haskey(handler.config.string_array_fn, :_all_)
         handler.config.string_array_fn[:_all_](nrows)
     else
-        if nrows <= 2 << 7 
+        if nrows + 1 < 2 << 7
```

Comment:
Do you need to add 1 here since you've already changed the operator to `<`?

Reply:
yes - it was deliberate

Reply:
yes.  both change were required .. I can add a file with 1 row, 254 rows and 256 row also if you want?

Reply:
@tk3369 is this Test sufficient?   I feel strongly it correct - we have production issues :-)